light_prop/algorithms.py

```diff
@@ -5,13 +5,14 @@
 @author: dr PK & be BB with mse MD
```

Comment:
@bbrozyna I see basically only whitespace changes here. I am wondering - is there a way to have a formatter preset for this repo/project?

Reply:
.[pre-commit -](https://pre-commit.com/) > sure I would do this, it is more important when there are more contributors

Reply:
https://github.com/bbrozyna/light-propagation/issues/15